src/lib/components/textarea.svelte

```diff
@@ -6,17 +6,18 @@
 	export let placeholder = ''
 	export let autofocus = false
 	export let disabled = false
+	export let readonly = false
 	export let pad = 0
-	export let rows = 1
+	export let height = 46
 
 	let placeholderHeight: number
 	let textarea: HTMLTextAreaElement
 
 	const resizeEvents = ['change']
-	const delayedResizeEvents = ['cut', 'paste', 'drop', 'keydown']
+	const delayedResizeEvents = ['change', 'cut', 'paste', 'drop', 'keydown']
 
 	function resize() {
-		textarea.style.height = 'auto'
+		textarea.style.height = height + 'px'
```

Comment:
```suggestion
		textarea.style.height = `${height}px`
```

It is recommended to use templating rather than string operations.